src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Stream.cs

```diff
@@ -111,8 +130,6 @@ private static async Task WriteAsyncCore<TValue>(Stream utf8Json, TValue value,
                     bufferWriter.Clear();
                 } while (!isFinalBlock);
             }
-
-            // todo: verify that we do want to call FlushAsync here (or above). It seems like leaving it to the caller would be best.
```

Comment:
any notes on this one?

Reply:
This was added during the original design\coding when we discussed a possible callback model (caller must call flush) instead of auto-flush. I don't believe we received any further feedback to add that capability.